client/src/services/deleteReview.js

```diff
@@ -0,0 +1,15 @@
+const deleteReview = async (userId, reviewId, reviewUserID) => {
```

Comment:
we don't need the front end to tell/ pass to the back end who the current user is - in the router we can use `req.user` to check with passport which user is currently stored in session